_posts/2022/2022-07-25-lessons-from-the-pivot.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Lessons From a Startup Pivot"
+description: "Depending on SAAS, PAAS, and IAAS is great, but it can hurt when you should have turned left at Albuquerque if you haven't insulated yourself from some of those choices."
+tags: [aspnetcore, azure, cloud]
+excerpt_image: https://user-images.githubusercontent.com/19977/180076597-6fbdb672-6539-43fd-b10d-1a1e667c8d8b.png
+---
+
+Building a startup is easy. You file some paperwork and bam! You're a startup!
+
+Building a start-up that's sustainable and can pay everyone a nice salary, on the other hand, is very tough. Today, as my company, [A Serious Business, Inc.](https://www.aseriousbusiness.com/) launches [Abbot on Product Hunt](https://www.producthunt.com/posts/abbot), I thought I'd reflect on some of the lessons learned in the past couple years.
```

Comment:
```suggestion
Building a start-up that's sustainable and can pay everyone a nice salary, on the other hand, is very tough. We are launching [Abbot on Product Hunt](https://www.producthunt.com/posts/abbot) today, and preparing last week made me reflect on some of the lessons we've learned since starting our business.
```
Feels like this could be shortened given the next line describes what Abbot is.